lib/gemstash/cli.rb

```diff
@@ -66,5 +67,20 @@ def status
     def stop
       Gemstash::CLI::Stop.new(self).run
     end
+
+    desc "preload", "Preloads all the gems in your gemstash server from the default upstream"
+    method_option :latest, :type => :boolean, :default => false, :desc =>
+      "Only fetch the latest specs"
+    method_option :server_url, :type => :string, :default => "http://localhost:9292", :desc =>
+      "Gemstash server url"
+    method_option :threads, :type => :numeric, :default => 20, :desc =>
+      "Number of threads to run the fetching job with"
+    method_option :limit, :type => :numeric, :default => nil, :desc =>
+      "Limit for the number of gems to preload"
+    method_option :skip, :type => :numeric, :default => 0, :desc =>
+      "Number of gems to skip when preloading"
+    def preload
```

Comment:
Yeah, but those could all be server side only, and probably make more sense that way anyways (importing a batch locally would probably be faster if you have the gems locally already, and then we can skip worrying about authorization keys).

Also, we have a lot of gem requirements that you really don't need if you are running client side stuff, so maybe a separate gem would make sense. However, that adds complexity, and the simplicity of just 1 gem is nice too.

Maybe we should get some advice from André?